masschallenge/django-accelerator
# AC-6406: Clean up existing Twitter handles

#### Changes introduced in [AC-6406](https://masschallenge.atlassian.net/browse/AC-6406)
- add migration to clean up the twitter handles on the platform

#### How to test
- checkout this branch and `make migrate`
- from the output, you will see that the twitter handles that have been transformed. They will be of the form
    - starts or ends with a `/`
    - has the handle in the form `*twitter.com/handle`
    - have `n/a` as a handle
    - have their handle as `https://twitter.`
    - have a valid handle with a leading `#`
    - have a hashbang at the beginning i.e. `#!/`
    - have a `lang` get variable on the twitter handle
